feature-detects/mediastream.js

```diff
@@ -0,0 +1,21 @@
+/*!
+{
+  "name": "MediaRecorder API",
+  "property": "mediarecorder",
+  "caniuse": "mediarecorder",
+  "tags": ["mediarecorder"],
+  "authors": ["Jaikishan Brijwani"],
+  "notes": [{
+    "name": "MDN Docs",
+    "href": "https://developer.mozilla.org/en-US/docs/Web/API/MediaStream_Recording_API"
+  }]
+}
+!*/
+/* DOC
+Detects support for the MediaStream Recording API for detecting mediastreaming.
+*/
+
+
+define(['Modernizr'], function (Modernizr) {
+    Modernizr.addTest('mediarecorder', typeof MediaRecorder);
+});
```

Comment:
You are still missing the `!== undefined` I told you earlier, that's what is causing the error. I post the image again (sorry I don't actually have the code anymore) so you can compare what's missing (if you click it shows it in big :)
![Code solution](https://user-images.githubusercontent.com/45839898/94997322-9bb76a00-05aa-11eb-96e4-a226244d26e2.png)